### PR TITLE
hw-mgmt: psu: Fix PSU eeprom vpd reading

### PR DESCRIPTION
**Pull request by sholeksandr**
Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

File: usr/usr/bin/hw-management-thermal-events.sh [hunk 1]
```diff
@@ -767,6 +767,7 @@ if [ "$1" == "add" ]; then
 		hw-management-ps-vpd.sh --BUS_ID "$bus" --I2C_ADDR 0x"$ps_ctrl_addr" --dump --VPD_OUTPUT_FILE $eeprom_path/"$psu_name"_vpd
 		if [ $? -ne 0 ]; then
 			# PS EEPROM VPD.
+            date
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management-thermal-events.sh:770]: What is the purpose of 'date' here?

**Comment by sholeksandr** [comment]
КУКГТ

**Comment by sholeksandr** [comment]
RERUN

**Review by vadimp-nvidia — APPROVED** [review]

**Comment by sholeksandr** [comment]
RERUN

**Pull request merged**